### DISCUSSION TITLE
Solving environment: failed HELP!

### DISCUSSION THREAD

**Discussion by airbender97** [jotech/gapseq#136] [Q&A]
Hello,

I am trying to install the GapSeq but I can't create an environment. 

> (base) C:\Users\user\gapseq\gapseq-1.2>conda env create -n gapseq-dev --file gapseq_env.yml
Collecting package metadata (repodata.json): done
Solving environment: failed

ResolvePackageNotFound:

  - sed
  - bedtools
  - exonerate
  - bioconda::libsbml
  - bc
  - blast
  - gawk
  - coreutils
  - bash
  - wget
  - hmmer
  - bioconductor-biostrings
  - grep

**Comment by Waschina** [comment]
Hi @airbender97 

I just tested the conda installation locally again, which worked without issues in solving the conda environment. My conda version is 4.11.0

Since I think the issue is more conda-related than gapseq-related, I would suggest to search for solutions of the `Solving environment: failed` in conda channels (e.g. [here](https://stackoverflow.com/questions/51266535/conda-returns-solving-environment-failed)).

**Reply by airbender97** [reply]
Hello @Waschina,

I solved the problem. I am an absolute new beginner in this kinda stuff. 

I didn't know that I need Linux/Ubuntu system to use these stuffs. I uploaded Ubuntu in Windows, which took almost 5 hours to solve the problems related to install Ubuntu :'). But anyways, both GapSeq and Ubuntu works very well now. Maybe you can inform new users about these kinda stuffs. 

Thank you!
